sdk/storage/Azure.Storage.Queues/src/QueueUriBuilder.cs

```diff
@@ -157,9 +165,9 @@ public QueueUriBuilder(Uri uri)
 
                 var startIndex = 0;
 
-                if (uri.IsHostIPEndPointStyle())
+                _isPathStyleUri = uri.IsHostIPEndPointStyle() || PathStylePorts.Contains(uri.Port);
```

Comment:
Is it worth moving this logic and set of ports into the IsHostIPEndPointStyle extension so it's shared?

Reply:
I will move it into `IsHostIPEndPointStyle()` but I will probably have to adjust this logic soon because we got a bug reported a while back about how specifying the ports caused a bug https://github.com/Azure/azure-sdk-for-net/issues/15693